interni/norme-di-progetto/components/processi-organizzativi/formazione-del-personale.tex

```diff
@@ -1,30 +1,34 @@
 \documentclass[../../norme-di-progetto.tex]{subfiles}
 
-% FIXME struttura non conforme
-% FIXME attività mancanti
 \begin{document}
 
-\subsubsection{Scopo}%
-\label{subs:formazione_del_personale/scopo}
+\subsubsection{Finalità}%
+\label{subs:formazione_del_personale/finalita}
 
 Il processo di formazione del personale prevede di avere sempre personale preparato, qualificato e all'altezza dei compiti da svolgere durante tutto il corso del progetto.
+Viene richiesto ad ogni componente del gruppo di impegnarsi in autonomia ad effettuare attività di autoapprendimento qualora il compito
```

Comment:
Qui c'è un a capo sospetto in mezzo alla frase

Reply:
li metto io per leggibilità; latex ignora gli a capo come se fosse html